o1vm/src/pickles/prover.rs

```diff
@@ -110,9 +111,17 @@ where
         };
         // Doing in parallel
         let scratch = scratch.into_par_iter().map(eval_col).collect::<Vec<_>>();
+        let scratch_inverse = scratch_inverse
+            .into_par_iter()
```

Comment:
Is this as efficient (or better) than having all of the scratch inverses in a single list and doing a single batch inversion of it all, instead of `n` parallel of shorter ones? Wondering for performance.

Reply:
I do not know. I'm wondering too. Might be good to investigate later. Opening a follow-up issue.

Reply:
Opened https://github.com/o1-labs/proof-systems/issues/2818